src/brevitas/core/function_wrapper/adaround_impl.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2023, Advanced Micro Devices, Inc. All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+
+
+"""
+Different implementations for adaround parameter transformation
+"""
+
+import torch
+import brevitas
+
+@brevitas.jit.script
```

Comment:
Will this function be compiled by the JIT if enabled?

Reply:
it can, but it wouldn't follow our structure. This should be turned into a brevitas ScriptModule with temperature as a constant.